bears/kotlin/KotlinLintBear.py

```diff
@@ -0,0 +1,26 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='ktlint',
+        global_bear=True,
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+):'
+                     r'(?P<column>\d+): (?P<message>.+)')
+class KotlinLintBear:
+    """
+    Lints your Kotlin files.
+    Checks for coding standards or semantic problems in Kotlin files.
+    """
+    LANGUAGES = {'kotlin'}
+    REQUIREMENTS = {DistributionRequirement(brew='shyiko/ktlint/ktlint')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    SEE_MORE = 'https://ktlint.github.io'
+
+    @staticmethod
+    def create_arguments(config_file):
+        return ()
```

Comment:
@anctartica, maybe you need to add a generate_config method and override the parameter `insert_final_newline` and then pass it to the create_arguments? [see](http://api.coala.io/en/latest/Developers/Linter_Bears_Advanced.html)

Reply:
@sks444 ktlint does not use a configuration file. It automatically detects an `.editorconfig` file if present in the current directory.

Reply:
what you are saying cannot be done

Reply:
so we don't specify a configuration file with ktlint on command line

Reply:
oh wow.  that calls for https://github.com/coala/coala/issues/4098 @satwikkansal